app/controllers/AddressLookupController.scala

```diff
@@ -65,7 +60,7 @@ class AddressLookupController @Inject()(journeyRepository: JourneyRepository, ad
     withJourneyV2(id) { journeyData =>
       val isWelsh = getWelshContent(journeyData)(req)
       val formPrePopped = lookupForm.fill(Lookup(filter, PostcodeHelper.displayPostcode(postcode)))
-      (Some(journeyData.copy(selectedAddress = None)), Ok(views.html.v2.lookup(id, journeyData, formPrePopped, isWelsh)))
+      (Some(journeyData.copy(selectedAddress = None)), Ok(views.html.v2.lookup(id, journeyData, formPrePopped, isWelsh = isWelsh)))
```

Comment:
Does it need to be passed in with the parameter name 'isWelsh='?
In the Edit controller, the parameter is just passed in.

Reply:
yes because of the order of the methods parameters.. this wasnt my story the parameters for the view are
isWelsh: Boolean,
message: Option[String] = None